devtools/platform-descriptor-json/src/main/resources/codestarts/amazon-lambda-example/codestart.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: amazon-lambda-example
+ref: amazon-lambda
+type: example
+missing-languages:
+  - kotlin
+  - scala
```

Comment:
FYI, they are now auto-detected, `missing-languages` is now (and already) obsolete. Sorry about this..